lib/rollbax.ex

```diff
@@ -46,20 +46,22 @@ defmodule Rollbax do
 
   @doc false
   def start(_type, _args) do
-    import Supervisor.Spec
-
-    enabled = get_config(:enabled, true)
-    custom  = get_config(:custom, %{})
-
-    token = fetch_and_resolve_config(:access_token)
-    envt  = fetch_and_resolve_config(:environment)
+    enabled = Application.get_env(:rollbax, :enabled, true)
+    custom = Application.get_env(:custom, %{})
+    environment = resolve_system_env(Application.fetch_env!(:rollbax, :environment))
+
+    access_token =
+      case enabled do
```

Comment:
Not an `if`? :)

Reply:
@ericmj `:enabled` can be `true | false | :log` so we need to know that it is `true` specifically :)